client/packages/host/src/Admin/ConfigurationSettings.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { BaseButton, useTranslation } from '@openmsupply-client/common';
+import { useName } from '@openmsupply-client/system';
+
+import { Setting } from './Setting';
+
+import { useConfigureNameProperties } from '../api/hooks/settings/useConfigureNameProperties';
+
+export const ConfigurationSettings = () => {
+  const { mutateAsync, isLoading } = useConfigureNameProperties();
+  const t = useTranslation('app');
+
+  const configure = async () => {
+    await mutateAsync();
+  };
+
+  const { data, isLoading: dataLoading } = useName.document.properties();
+
+  const propertiesAlreadyConfigured = !!data?.length;
```

Comment:
I think we might want to do a patch and update these settings.
Do you think we should allow people to re-run it? Or implement a better check?

Reply:
I think rerunning is fine since it does an upsert, but probably nice to also show some kind of indication that the properties have at least been configured at some point?

(As discussed in person, definitely can be improved how we manage these, probably would want a more fleshed out UI sometime soon 👍)